skin/frontend/base/default/js/somethingdigital/back-to-top.js

```diff
@@ -1,12 +1,18 @@
 (function($) {
   var BackToTop = {
     scrollAppears: 300,
-    scrollSpeed: 300,
+    scrollSpeed: BackToTopConfig.speed,
+    persistMethod: BackToTopConfig.persist,
     anchorSide: 'left',
```

Comment:
Is this not configurable? Clintons technically has desktop design set to right side.